src/compiler/transformers/classFields.ts

```diff
@@ -622,9 +775,14 @@ namespace ts {
                 properties = filter(properties, property => !!property.initializer || isPrivateIdentifier(property.name));
             }
 
+            const privateMethods = filter(

```

Comment:
Is this really needed? `elements` already seems to contain all private methods props and accessors ..

Reply:
Not sure what `elements` refer to here.

Reply:
Ah my bad. I was looking at part of the method above that has an elements.